_data/companies/movus.yml

```
@@ -1,6 +1,6 @@
 name: MOVUS
-wfh: Required
-travel: Restricted
+wfh: Required (permission required to NOT work from home)
```

```suggestion
wfh: Required unless permission given otherwise
```

What about that as a way to shorten it?

---

Good idea. Reads better. Thanks.